src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
@ljharb I've just checked it and if you use Babel's `browser-polyfill.js` (which is mostly compiled [core-js](http://npmjs.com/package/core-js) & [regenerator](https://www.npmjs.com/package/regenerator)) then the global `Symbol` is defined but `typeof Symbol.iterator === "object"`. I think it's good to work in such an environment as well.

I should probably add a test for that.